.github/workflows/release.yaml

```diff
@@ -43,4 +62,66 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: build/release.sh $GH_TAG
+  
+  # # # # # # # # # # # # # # # # #
+  # Update Helm Chart
+  # # # # # # # # # # # # # # # # #
+  update-helm-chart-version:
+    name: Update Helm Chart Version
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Update Chart
+        # https://github.com/marketplace/actions/github-push
+        run: |
+          CAPTURE_GROUP="appVersion:\ \"v[0-9]\+\.[0-9]\+\.[0-9]\+\""
+          REPLACEMENT="appVersion\:\ \"$GH_TAG\""
+          echo "Updating Piko Helm Chart to Version $GH_TAG"
+          sed -i'' "s/$CAPTURE_GROUP/$REPLACEMENT/" $GITHUB_WORKSPACE/operations/helm/piko/Chart.yaml
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git commit -am "[bot] bump helm chart to version $GH_TAG"
+          git push origin HEAD:main
 
+  # # # # # # # # # # # # # # # # #
+  # Build & Publish Containers
+  # # # # # # # # # # # # # # # # #
+  docker-containers:
+    name: Build & Publish Containers
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v4
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to GHCR
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do I need to configure anything for this to work?

Reply:
No I don't think there is anything you need to do. `GITHUB_TOKEN` is provided by the runner and `repository_owner` is just a metadata object. I have not configured any secrets in my fork and it works fine.